libs/core/src/stories/getting-started.docs.mdx

```diff
@@ -0,0 +1,96 @@
+import { Meta, Story } from '@storybook/blocks';
+
+<Meta title="Resources/Get Started"/>
+
+# Getting Started
+
+Pine is a library of Web Components crafted by the Design System Services team at Kajabi. It provides a robust foundation for building cohesive and accessible user interfaces. Designed with scalability and performance in mind, Pine enables developers and designers to collaborate seamlessly on consistent UI experiences.
+
+## Repository
+
+The source code can be found in the [Github Repo](https://github.com/Kajabi/pine).
+
+## Design Files
+
+The component designs can be found in the [Figma Project](https://www.figma.com/design/CC1YmaGKHnsvB28yLY9mEH/%E2%9D%96-Mercury-components?m=auto&node-id=0-1&t=xybx2d1s26U4p0Jd-1).
+
+## Initial Setup
+
+Follow these steps to set up the repository locally:
+
+1. Clone the repository:
+```bash
+git clone git@github.com:Kajabi/pine.git
+```
+
+2. Move to the repository:
+```bash
+cd pine
+```
+
+3. Install the required dependencies:
+```bash
+brew install nss  # Required if using Firefox
+mkcert -install   # Add mkcert to trusted CAs
+npm run setup     # Use the correct Node.js version
+```
+
+4. Generate a local SSL certificate:
+```bash
+cd libs/core
+mkcert -cert-file pineLocalDev.pem -key-file pineLocalDev-key.pem localhost 7300
+```
+
+5. Start the development server:
+```bash
+cd ../../ # Return back to the root directory
+npm start
+```
+
+> Note: If starting the development server process hangs, restart it.
+
+## Generating Components
+
+1. Move to the core folder:
+```bash
+cd libs/core
+```
+
+2. Generate component
+```bash
+npm run stencil.generate pds-[component-name]
+```
+
+3. Choose `*.scss` format when prompted.
+
+4. Confirm all folders and files are aligned with other preexisting components.
+
+
+## Project Setup
+The project is divided into two folders `libs/core` and `libs/react`.
+
+### core
+The `core` folder houses all web component files. This is where we build new components in StencilJS.
+
+### react
```

Comment:
```suggestion
### React
```

Reply:
this is referring to the folder name which is lowercase